src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Members.cs

```diff
@@ -339,11 +350,15 @@ public override void VisitMethod(IMethodSymbol symbol)
             {
                 case MethodKind.Ordinary:
                 case MethodKind.DelegateInvoke:
-                case MethodKind.ReducedExtension:
                 case MethodKind.LocalFunction:
                     //containing type will be the delegate type, name will be Invoke
                     builder.Add(CreatePart(SymbolDisplayPartKind.MethodName, symbol, symbol.Name));
                     break;
+                case MethodKind.ReducedExtension:
+                    // Note: Extension methods invoked off of their static class will be tagged as methods.
+                    //       This behavior matches the semantic classification done in NameSyntaxClassifier.
```

Comment:
very minor.  But it is a bit strange for this comment to be here.  basically, it's the compiler saying: here' show the IDE does it :)   It's not actually a probem, but i mentally think about things the reverse way. 


--

you don't need ot change anything here :)